tests/test_core.py

```diff
@@ -2017,7 +2016,8 @@ def test_memorygrowth_3(self):
       self.skipTest('test needs to modify memory growth')
 
     # checks handling of malloc failure properly
-    self.emcc_args += ['-s', 'ALLOW_MEMORY_GROWTH=0', '-s', 'ABORTING_MALLOC=0', '-s', 'SAFE_HEAP']
+    self.set_setting('ABORTING_MALLOC', 0)
```

Comment:
```suggestion
    self.clear_setting('ABORTING_MALLOC')
    self.clear_setting('ALLOW_MEMORY_GROWTH')
```

Reply:
ABORTING_MALLOC defaults to '1' so clearing is not enough, it needs to be explicitly set to 0

Reply:
Oh I see, I was miunderstanding what `clear_setting` meant.